edb/edgeql/ast.py

```diff
@@ -266,22 +265,17 @@ def from_python(cls, val: typing.Any) -> BaseConstant:
 
 
 class StringConstant(BaseConstant):
-    quote: str
 
     @classmethod
     def from_python(cls, s: str):
-        s = s.replace('\\', '\\\\')
-        value = quote.quote_literal(s)
-        return cls(value=value[1:-1], quote="'")
+        return cls(value=s)
 
 
 class RawStringConstant(BaseConstant):
-    quote: str
 
     @classmethod
     def from_python(cls, s: str):
-        value = quote.quote_literal(s)
-        return cls(value=value[1:-1], quote="'")
+        return cls(value=s)
```

Comment:
This node should probably be dropped altogether as there's no more distinction between this and `StringConstant`.